site/content/en/docs/tasks/run/kubeflow/jaxjobs.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Run a JAXJob"
+date: 2025-04-23
+weight: 6
+description: >
+  Run a Kueue scheduled JAXJob
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Training Operator](https://www.kubeflow.org/docs/components/training/jax/) JAXJobs.
```

Comment:
```suggestion
This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Trainer](https://www.kubeflow.org/docs/components/training/jax/) JAXJobs.
```

The training-operator was rebranded as `trainer`. So, could you replace all with `trainer` in this doc?